build_test/lib/src/in_memory_reader.dart

```diff
@@ -100,65 +79,38 @@ class InMemoryAssetReaderWriter extends AssetReader
         .where((id) => id.package == package && glob.matches(id.path)));
   }
 
-  @override
-  void cacheBytesAsset(AssetId id, List<int> bytes) {
-    assets[id] = bytes;
-  }
-
-  @override
-  void cacheStringAsset(AssetId id, String contents, {Encoding? encoding}) {
-    encoding ??= utf8;
-    assets[id] = encoding.encode(contents);
-  }
-
   @override
   Future writeAsBytes(AssetId id, List<int> bytes) async {
-    assets[id] = bytes;
+    assets[id] = Uint8List.fromList(bytes);
```

Comment:
maybe `bytes is Uint8List ? bytes : Uint8List.fromList(bytes);`?
In general I think the interface should be changed, but probably not in this cl...

Reply:
Done, thanks.

This is generator-facing API so we can't easily change it.